src/navigator/Navigator.tsx

```diff
@@ -757,6 +758,7 @@ function RootStackScreen() {
         // take up the whole screen, it is no longer obvious that they are a bottom
         // sheet / how to navigate away
         maxDynamicContentSize: variables.height * 0.9,
+        backgroundStyle: { backgroundColor: Colors.backgroundPrimary },
```

Comment:
should we also add the handleStyle here similar to BottomSheetBase so that is consistent?

Reply:
oooooh yes i missed this!!